### ISSUE TITLE
Publish handler for null sent no ready signal

### ISSUE THREAD

**Issue by derwaldgeist** [3stack-software/meteor-presence#4]
Since I added this package and set-up a publish function, I'm getting this error message:

```Publish handler for null sent no ready signal```

Here's my publish function:

```js
/* global presences */

Meteor.publish('userPresence', function(options) {
  options = options || {};
  _.defaults(options, {
    users: []
  });
  check(options, {
    users: [String],
    session: Match.Optional(String)
  });

  var selector = {
    _id: {
      $ne: this.connection.sessionKey // don't publish the current user
    },
    userId: {
      $in: options.users
    },
    status: 'online' // publish only clients that called 'setPresence'
  };
  if (options.session) {
    selector['state.session'] = options.session;
  }
  var cursor = presences.find(selector, {
    fields: {
      state: true,
      userId: true
    }
  });
  return cursor;

});
```
The ```state.session``` property is a custom state extension which is not always present, so I set it up like this:
```js
/* global Presence */
Presence.configure({
  state: function() {
    return {
      online: true,
      session: undefined
    };
  }
});
```

**Comment by derwaldgeist** [comment]
Some googling lead me to this thread:
https://github.com/kadirahq/fast-render/issues/12

As described there, this error message appears if a null publication does not send this.ready() within 500ms. I think this is caused by this publication:

https://github.com/3stack-software/meteor-presence/blob/master/lib/server/presence.coffee#L26

**Comment by nathan-muir** [comment]
Hey @derwaldgeist , if you go 24 lines down, you can see the call to `@ready()` - 
https://github.com/3stack-software/meteor-presence/blob/master/lib/server/presence.coffee#L50

Make sure you're using `3stack:presence@1.0.4`

**Comment by derwaldgeist** [comment]
Hi nathan-muir, thanks for your quick replay. Yes, I saw the @ready() statement, but it seems as if the code before that needs more than 500ms, so the warning is logged despite of that. BTW: I am not able to use 1.0.4. If I try, I get a package dependency error stating that 1.0.4 is not satisfied by 1.0.3. This happens although I only have packages referencing 1.0.4. Only 1.0.3 worked fine for me. Maybe there's a problem with the Atmosphere publication of this package? I'm not familiar with the package publication details, I can just explain the symptoms...

**Comment by nathan-muir** [comment]
Ah, well the fix was only in the 1.0.4 version.

Are you using version 1.2 of Meteor?

**Comment by derwaldgeist** [comment]
Thanks for the 1.0.5 package, I could successfully upgrade to that version! As far as I can see it, works like a charm now - and no error messages anymore. Once again, thank you for your quick support!
